src/cast.c

```diff
@@ -0,0 +1,271 @@
+#include "vctrs.h"
+
+// Initialised at load time
+static SEXP vec_cast_dispatch_fn = NULL;
+
+
+static SEXP int_as_logical(SEXP x, bool* lossy) {
```

Comment:
Why is this static?

Reply:
To indicate these are support functions for other functions in this file. We can remove the static if we add these functions in vctrs.h.

Reply:
Ah got it.